src/pages/OrganizationDetail.tsx

```diff
@@ -3,11 +3,18 @@ import { useParams } from 'react-router-dom';
 import { withOidcSecure } from '@axa-fr/react-oidc';
 import { Organization } from '@pdc/sdk';
 import { PanelGrid, PanelGridItem } from '../components/PanelGrid';
+import { mapFieldNames } from './ProposalList';
+import { mapProposals } from '../map-proposals';
 import {
 	useOrganization,
 	useOrganizations,
 	ORGANIZATIONS_DEFAULT_PAGE,
 	ORGANIZATIONS_DEFAULT_COUNT,
+	useBaseFields,
+	useProposals,
+	PROPOSALS_DEFAULT_COUNT,
+	PROPOSALS_DEFAULT_PAGE,
+	PROPOSALS_DEFAULT_QUERY,
```

Comment:
I just realized there's a _third_ place we're going to want a proposal table before long: [on the detail page for a bulk upload](https://github.com/PhilanthropyDataCommons/front-end/issues/513). This will be a Phase 3 page, but it's worth knowing it's coming. (These proposals, FWIW, will additionally filter by bulk upload ID.)